examples/ibm-is-ng/main.tf

```diff
@@ -584,12 +584,14 @@ resource "ibm_is_instance" "instance4" {
 resource "ibm_is_snapshot" "b_snapshot" {
```

Comment:
Check  this folder examples->common->github.com/IBM -> there is vpc-go-sdk

Reply:
removed